sympy/utilities/runtests.py

```diff
@@ -32,7 +34,6 @@
 import signal
 import stat
 import tempfile
-
```

Comment:
Here you have deleted a line from the file. I'm not sure if that was intentional but you should try to avoid making unrelated changes when sending a pull request. You can check the diff either before pushing or on github to see what changes you have made. Did you mean to delete this line?

Reply:
Sorry about that. I might have deeted it unintentionally because it was an empty line. I will surely keep this in mind in the future.